docs/Halogen-Events.md

```diff
@@ -64,6 +64,14 @@ This function provides an alternative to making two nested calls to `pure`:
 onclick (input \_ -> Input)
 ```
 
+#### `withEventHandlerT`
+
+``` purescript
+withEventHandlerT :: forall i m e. (e -> EventHandlerT m i) -> e -> EventHandler (m i)
```

Comment:
Well, `Attr` is defined in terms of `EventHandler` (otherwise would need another `m` type argument), and the smart constructors for event handlers (`onclick` etc.) therefore require `EventHandler (m i)`. If you want to use `EventHandlerT`, this is just here as a helper function. Another option is that we change things like `onclick` to use `EventHandlerT`.